README.md

```diff
@@ -23,10 +23,44 @@
 
 <p><img src="schedule.png" alt="LTS Schedule"/></p>
 
-The LTS Schedule is available also as a [JSON][] file or [ICal][]. There is
```

Comment:
`LTS working group` is referred to above, I assume that should be either the `Release working group` or the `LTS team`.